heidegger_index/templates/markdown/work_detail.md

```diff
@@ -0,0 +1,12 @@
+# Index of {{ work.csl_json.title }}
+
+## Work reference
+{{ work.reference_md | safe }}
+
+## Index terms
+{% include 'markdown/_group_by_lemma.md' with ref_list=term_list %}
+{% if person_list %}{% include 'markdown/_group_by_lemma.md' with ref_list=person_list group_title="People mentioned" %}{% endif %}
+{% if work_list %}{% include 'markdown/_group_by_lemma.md' with ref_list=work_list group_title="Works mentioned" %}{% endif %}
+
+Source:
+André van Delft and Johannes de Wit, _{{ work.title }} – Heidegger index_ (2022) \<<https://heidegger.delve.nu{% url 'index:work-detail' work.slug %}>\>.
```

Comment:
Misschien deze refactoren in een aparte file? En i.p.v. die hard-coded '2022' kun je gebruik maken van de `now` template tag: https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#now. Misschien kun je zelfs de hele datum noemen ('geraadpleegd op').

Reply:
Dat is precies wat ik gedaan heb!